### PR TITLE
Add Abbas Zaidi to Before I Die

### PR DESCRIPTION
**Pull request by AbbasZaidi11**
I am adding Abbas Zaidi to Before I Die with images and text.

### CODE REVIEW

File: src/Components/LocationMap/LocationMap.jsx
Comment (XanderRubio) [inline src/Components/LocationMap/LocationMap.jsx:1]: Hello @AbbasZaidi11, thank you for your contribution. 

Could you please assist me by providing details of the changes or additions you made in the LocationMap.jsx file? As your initial pull request for adding "Before I Die" only involved four files, and this is an additional code related to enhancement, I would appreciate it if you could explain the additions you made for my understanding before I merge your request. 

Also, I want to compliment you on the great image and text you submitted as they show up great on the preview deployment. You will create the opportunity for yourself to travel to New York. Thank you!

Comment (AbbasZaidi11) [reply src/Components/LocationMap/LocationMap.jsx:1]: Sorry, if there is any sort of confusion but I did not make any changes to the map component. Also thanks for letting me contribute.

Comment (XanderRubio) [reply src/Components/LocationMap/LocationMap.jsx:1]: Thank you for clarifying. When I reviewed your changes, I noticed some modifications related to the formatting of the file. I assume this occurred due to automatic formatting. The component works normally when viewing the preview deployment and I will double-check everything to ensure that everything is fine. Your contribution is fantastic, and I will proceed to merge it onto the site.

**Comment by XanderRubio** [comment]
@AbbasZaidi11 Your pull request has been successfully merged!!! Congratulations Bottle with Popping CorkBottle with Popping CorkBottle with Popping CorkDouble Exclamation MarkDouble Exclamation MarkDouble Exclamation Mark. Make it happen for venturing forth and visiting the cosmopolitan city of New York!

 If you haven't already, please go ahead and star Glowing StarGlowing StarGlowing Starthis repo and you are more than welcome to continually contribute to this project by assisting with our [open issues](https://github.com/BeforeIDieCode/BeforeIDieAchievements/issues), or adding your own new open issues, and additionally you can get an idea of ideas we have to continually grow this project further as technology and tech stack continually evolve by looking at the [ROADMAP](https://github.com/BeforeIDieCode/BeforeIDieAchievements/blob/main/ROADMAP.md) for the project. Have a great day, and thank you for taking the time to contribute, Abbas! You can view your contribution now on the [live link](https://before-i-die-achievements.vercel.app/). Have a great rest of your day and I will be sending you an invite to be a member of the Before I Die Code now.

**Pull request merged**